django/core/validators.py

```diff
@@ -66,22 +66,15 @@ def __eq__(self, other):
 
 
 @deconstructible
-class URLValidator(RegexValidator):
+class DomainNameValidator(RegexValidator):
+    message = _("Enter a valid domain name.")
     ul = "\u00a1-\uffff"  # Unicode letters range (must not be a raw string).
-
-    # IP patterns
-    ipv4_re = (
-        r"(?:0|25[0-5]|2[0-4][0-9]|1[0-9]?[0-9]?|[1-9][0-9]?)"
-        r"(?:\.(?:0|25[0-5]|2[0-4][0-9]|1[0-9]?[0-9]?|[1-9][0-9]?)){3}"
-    )
-    ipv6_re = r"\[[0-9a-f:.]+\]"  # (simple regex, validated later)
-
-    # Host patterns
+    # Max length for domain name labels is 63 characters per RFC 1034 sec. 3.1
```

Comment:
The comment `# Max length for domain name labels is 63 characters per RFC 1034 sec. 3.1` should be above `domain_re` 
I also think as this is best placed above the regex, we can remove the duplicate comment above `domain_re = DomainNameValidator.domain_re` in `URLValidator`
With a similar logic, add the `# Host patterns` comment here and remove the `# Host patterns` comment from above `hostname_re = DomainNameValidator.hostname_re` in `URLValidator`

Let's also add full stops to the end of the comments which are not inline (so `# Host patterns.` but maybe don't bother with the `# can't end with a dash` comments)

Reply:
Thanks - comments in DomainNameValidator updated and relevant comments in URLValidator.